### PR TITLE
Remove mention of XMPP bridge, we don't have one

### PR DESCRIPTION
**Pull request by dawidpotocki**
… and it's not that "New", I guess.

https://deploy-preview-1246--privacytools-io.netlify.com/contact/

### CODE REVIEW

**Comment by dawidpotocki** [comment]
> I would like to keep it, as I have heard that they are finally planning on restoring it. Upstream issue: https://github.com/matrix-org/matrix-bifrost/issues/73

There is no info in this issue. If it will come back, we can just readd
it. For now it is misleading.

File: contact.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ It's very important to us to stay up-to-date on the latest changes in the privac
 
 For complete transparency, software and providers will only be considered for this website after discussions take place on either of the above platforms.
 
-<span class="text-success"><i class="fas fa-comment"></i> New:</span> We're now on Matrix! Join us at `#general:privacytools.io` to chat with us and other members about this site and privacy in general. If you need a Matrix account, you can sign up with our own homeserver (`https://chat.privacytools.io`) using [riot.privacytools.io](https://riot.privacytools.io/) or any Matrix client! If you prefer XMPP, you're welcome to [use the Matrix.org XMPP bridge to join the discussion](https://conversations.im/j/%23general%23privacytools.io@bridge.xmpp.matrix.org).
+We're now on Matrix! Join us at `#general:privacytools.io` to chat with us and other members about this site and privacy in general. If you need a Matrix account, you can sign up with our own homeserver (`https://chat.privacytools.io`) using [riot.privacytools.io](https://riot.privacytools.io/) or any Matrix client!
```
Comment (Mikaela) [inline contact.md]:
```suggestion
We're on Matrix! Join us at `#general:privacytools.io` to chat with us and other members about this site and privacy in general. If you need a Matrix account, you can sign up with our own homeserver (`https://chat.privacytools.io`) using [riot.privacytools.io](https://riot.privacytools.io/) or any Matrix client!
```

It's not that new is it?

Comment (Mikaela) [reply contact.md]: And should there be a reference to https://github.com/privacytoolsIO/privacytools.io/issues/1049 ?

Comment (dawidpotocki) [reply contact.md]: > And should there be a reference to #1049 ?

That should be in different PR.

**Comment by moppman** [comment]
Not to get your hopes up, but maybe...https://nitter.net/matrixdotorg/status/1181257728678866950

**Comment by Mikaela** [comment]
The most recent on Bifröst that I have heard on GItHub is https://github.com/privacytoolsIO/privacytools.io/issues/1389#issuecomment-540834187 and https://matrix.to/#/!vKsKNErebdRaQvsUkQ:half-shot.uk/$kadAGSTq93z8hu4aRvomq9-067BrE09j0nbN2nWuI-k?via=half-shot.uk&via=matrix.org&via=privacytools.io 

> I think it's unlikely that it will happen before 21st now, because of company timings.

In case the link above doesn't work for you, it's from `#bifrost:privacytools.io` / `#bifrost:half-shot.uk`.

**Comment by moppman** [comment]
The official matrix<-> xmpp bridge is now up again, running on matrix.org.
See https://github.com/matrix-org/matrix-bifrost/issues/73#issuecomment-579679809

**Pull request merged**